packages/UI/CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# Getting started with the Platform Enablement Team
+
+dev-launchers-platform is a mono repo using yarn. All yarn script lines need to be run in the root folder. 
+
+## Installing and quick started
+1. install yarn, below are several ways to install. 
+    - https://classic.yarnpkg.com/en/docs/install#windows-stable
+    - In a terminal with admin run ```$ corepack enable```
+    - ```$ npm install --global yarn```
+2. clone the repo @  https://github.com/dev-launchers/dev-launchers-platform
+3. After cloning repo, navigate to /dev-launchers-platform
+4. ```$ yarn install``` in the root folder to install dependencies 
+5. Open a new terminal and run ```$ yarn workspace @devlaunchers/tailwind dev```. to start tailwind constructor.
+5. Open a 2nd terminal and run ```$ yarn workspace @devlaunchers/components storybook``` to start storybook.
+7. Start Development work.
+
+## Yarn scripts basics.
+
+This is just a quick rundown on how to run scripts in a mono repo using yarn, you will need to target a workspace with the following CLI.  Further information can be found in the root folder markdowns. 
+
+    $ yarn workspace @devlaunchers/<workspace> <cmd>
+
+### Yarn scripts for this team
+```
+$ yarn workspace @devlaunchers/tailwind dev
+$ yarn workspace @devlaunchers/components storybook
+$ yarn workspace @devlaunchers/components typecheck
+```
+More scripts are located in the package.json file. 
+
+### Installing packages
+
+To install a package into a specific workspace:
+
+    $ yarn workspace @devlaunchers/<workspace> add <library>
+
+#### Installing Radix-ui components
+
+Radix-ui components need to be installed individually, if you are not able to import @radix-ui follow the command below to install in devlaunchers components workspace.
+
+    $ yarn workspace @devlaunchers/components add <library>
+
+Below is the command to install radix-ui/react-checkbox
+
+    $ yarn workspace @devlaunchers/components add @radix-ui/react-checkbox
+
+
+## Git branch checkout flow
+
+We will be working out of development/components. 
+
+    $ git checkout development/components
+
+- after switching to the development/components branch, make sure you "yarn install" for new dependencies
+
+### Out of "development/components" branch all new branches will be created off of it.
+
+    $ git checkout -b components/newBranchName
+
+
+## Figma
+
+Go to universal Figma @:
+
+    https://www.figma.com/file/EwzuhhvTulvFRMvhTD5VAh/DL-Universal-Design-System?type=design&t=w1PSPgQunxcd19Bz-6
+
+- All component designs will be coming from the universal UI library.
+- Make sure dev mode is toggled in Figma.
+- All tailwind classes will be made up of dashes, other characters like '/' will be replaced with '-'.
+
+## Vscode extensions
+
+1. Tailwind css intellisense
+2. Eslint
+3. ES7 + React/redux/React-native snippets.
+4. Figma for vs code
```

Comment:
I like these sections alot, maybe we can add a toast in vscode to ask developers to install extensions like in here https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions